pkg/util/mount/mount_windows.go

```diff
@@ -74,8 +74,23 @@ func (mounter *Mounter) Mount(source string, target string, fstype string, optio
 			return nil
 		}
 
-		// empty implementation for mounting azure file
-		return os.MkdirAll(target, 0755)
+		cmd := fmt.Sprintf(`$User = "AZURE\%s";$PWord = ConvertTo-SecureString -String "%s" -AsPlainText -Force;`+
+			`$Credential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList $User, $PWord`,
+			options[0], options[1])
+
+		driverLetter, err := getAvailableDriveLetter()
+		if err != nil {
+			return err
+		}
+		bindSource = driverLetter + ":"
+		cmd += fmt.Sprintf(";New-SmbGlobalMapping -LocalPath %s -RemotePath %s -Credential $Credential", bindSource, source)
+
+		if output, err := exec.Command("powershell", "/c", cmd).CombinedOutput(); err != nil {
+			// we don't return error here, even though New-SmbGlobalMapping failed, we still make it successful,
+			// will return error when Windows 2016 RS3 is ready on azure
+			glog.Errorf("windowsMount: SmbGlobalMapping failed: %v, output: %q", err, string(output))
+			return os.MkdirAll(target, 0755)
+		}
 	}
 
 	if output, err := exec.Command("cmd", "/c", "mklink", "/D", target, bindSource).CombinedOutput(); err != nil {
```

Comment:
want to make sure,  `cmd` here is a windows command, not the `cmd` variable defined above?

Reply:
yes, and this "cmd" var name would be error prone, I have changed the name to cmdLine